packages/design-system/src/components/Tooltip/Tooltip.tsx

```diff
@@ -160,6 +182,13 @@ export const Tooltip = (props: TooltipProps) => {
       props.onOpen && props.onOpen();
     } else {
       props.onClose && props.onClose();
+
+      // if tooltip goes from active to inactive and is the dialog version, focus the trigger
+      if (prevActiveStateVar && (props.dialog || isMobile) && props.showCloseButton) {
```

Comment:
Does the `props.showCloseButton` part matter?

Reply:
I guess I did that because _technically_ a tooltip could be `dialog='true'` but not have a close button. So I'm not sure if the trigger should be focused on when the user clicks elsewhere in the document

Reply:
That's fair. I doubt anyone will complain about those two behaviors being inconsistent because no product will have two different configurations side by side.